### PR TITLE
BLOCKED until 2.0 is released - Feature - Export scenario

### PR DESCRIPTION
**Pull request by khaitruong922**

### CODE REVIEW

File: docs/manuals/core/entities/scenario-cycle-mgt.md
Comment (jrobinAV) [inline docs/manuals/core/entities/scenario-cycle-mgt.md]:
```suggestion
```python linenums="1"
import taipy as tp
import my_scenario_config

scenario = tp.create_scenario(my_scenario_config)
tp.export(scenario.id, "./my_scenario")
```
```

**Review by jrobinAV — APPROVED** [review]

File: docs/manuals/core/entities/scenario-cycle-mgt.md
Comment (FabienLelaquais) [inline docs/manuals/core/entities/scenario-cycle-mgt.md:138]: Just make sure section titles are homogeneous.
I can see 'Cycle attributes' below.
This then should be 'Scenario exports'.

Often something like 'Exporting scenarios' is used...

**Review by FabienLelaquais — APPROVED** [review]

File: docs/manuals/core/entities/scenario-cycle-mgt.md
Comment (khaitruong922) [reply docs/manuals/core/entities/scenario-cycle-mgt.md:138]: This is to make it consistent with headers above: Delete a scenario, Tag/untag a scenario, etc.

**Review by jrobinAV — APPROVED** [review]
Good catch Giang. 
Let s wait for the doc release before merging it, but it is good for me.

**Pull request merged**
